va/compat_win32.h

```diff
@@ -46,6 +46,15 @@ typedef LONG NTSTATUS;
 
 typedef unsigned int __uid_t;
 
+#if _MSC_VER
+#define getenv _getenv
+char _getenv_buf[32768];
```

Comment:
This will define `_getenv_buf` in each .c file which includes this header. And that might lead to multiple definitions of same variable, i.e. link error. Can you move that to function implementation and make static?